src/index.js

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- I would advise you to solve the Linter manually. Do not disable linters because it contributes to clean code, and avoid bugs in your code.

Reply:
Thanks for the insight fixed.